helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -246,6 +246,29 @@ receivers:
   {{- if .Values.logsCollection.extraFileLogs }}
   {{- toYaml .Values.logsCollection.extraFileLogs | nindent 2 }}
   {{- end }}
+
+  # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/journaldreceiver
+  {{- if (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}
+  {{- if and (eq .Values.logsEngine "otel") .Values.logsCollection.journald.enabled }}
```

Comment:
This whole block is under the following condition:
`{{- if and (eq .Values.logsEngine "otel") .Values.logsCollection.containers.enabled }}` 
starting from line 113. 

It means that if `.Values.logsCollection.containers.enabled=false`, this block will be excluded regardless of `Values.logsCollection.journald.enabled`.

Please update line 113 to 
```smarty
  {{- if and (eq (include "splunk-otel-collector.logsEnabled" .) "true") (eq .Values.logsEngine "otel") }}
  {{- if .Values.logsCollection.containers.enabled }}
```

Add the following line after line 244
```smarty
  {{- end }}
```

and here we should have only one line instead:
```smarty
  {{- .Values.logsCollection.journald.enabled }}
```

Reply:
Thanks @dmitryax. I'll update it. 
I think in this case we might need a separate pipeline for journald? 
Journald is not dependent on `{{- if .Values.logsCollection.containers.enabled }}` 
And I am guessing `extraFiles` is not dependent on `{{- if .Values.logsCollection.containers.enabled }}`  either.

Please let me know if otherwise.

Reply:
This why we need to add `{{- end }}` after line 244 to close the second `{{- if .Values.logsCollection.containers.enabled }}` condition and keep the first one

Reply:
Got it. I'll update this with other suggestions in the PR. thank you!